### PR TITLE
chore: Add a note for db/migrate and docker-env

### PR DESCRIPTION
**Pull request by igarashitm**
Cc: @dhirajsb

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ A service for provisioning and managing fleets of connector instances.
     ```
     make db/setup && sleep 1 && make db/migrate
     ```
+   > **NOTE**: `make db/migrate` might fail with `SIGSEGV` if your environment points to minikube docker-env. Make sure you haven't done such as `eval $(minikube docker-env)`.
```
Comment (dhirajsb) [inline README.md:89]: Change:
`Make sure you haven't done such as `eval $(minikube docker-env)`.`
to:
`Make sure you haven't run the command `eval $(minikube docker-env)`.`

**Review by dhirajsb — CHANGES_REQUESTED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (igarashitm) [reply README.md]: Done - thanks for the review :+1:

**Review by valdar — APPROVED** [review]

**Comment by igarashitm** [comment]
@dhirajsb or @valdar can you merge it? I don't have write access :smile:

**Pull request merged**